src/api.v2/helpers/pipeline/gem2s.js

```diff
@@ -19,17 +19,35 @@ const logger = getLogger('[Gem2sService] - ');
 
 const hookRunner = new HookRunner();
 
-const saveProcessingConfigFromGem2s = ({ experimentId, item }) => {
-  logger.log('Saving processing config for gem2s');
-  new Experiment().updateById(experimentId, { processing_config: item });
-  logger.log('Finished saving processing config for gem2s');
-};
+const continueToQC = async (payload) => {
+  const { experimentId, item } = payload;
+
+  await new Experiment().updateById(experimentId, { processing_config: item.processingConfig });
+
+  logger.log(`Experiment: ${experimentId}. Saved processing config received from gem2s`);
+
+  logger.log(`Experiment: ${experimentId}. Starting qc run because gem2s finished successfully`);
+
+  // we need to change this once we rework the pipeline message response
+  const authJWT = payload.authJWT || payload.input.authJWT;
+
+  const { stateMachineArn, executionArn } = await createQCPipeline(experimentId, [], authJWT);
+
+  await new ExperimentExecution().upsert(
+    {
+      experiment_id: experimentId,
+      pipeline_type: 'qc',
+    },
+    {
+      state_machine_arn: stateMachineArn,
+      execution_arn: executionArn,
+    },
+  );
 
-const continueToQC = () => {
+  logger.log('Started qc successfully');
 };
 
 hookRunner.register('uploadToAWS', [
-  saveProcessingConfigFromGem2s,
   continueToQC,
```

Comment:
It would be nice if you renamed `gem2sResponse` to `handleGem2sResponse`, to make the naming consistent with gem2s. Same with `gem2sCreate` and `createQCPipeline`.

Reply:
Sounds good, I did a lot of copy pasting without a lot of refactoring